Source/DafnyCore/Rewriters/OpaqueMemberRewriter.cs

```diff
@@ -140,11 +140,11 @@ public OpaqueMemberRewriter(ErrorReporter reporter)
     }
     Method reveal;
     if (m is TwoStateFunction) {
-      reveal = new TwoStateLemma(m.RangeToken, m.NameNode.Prepend("reveal_"), m.HasStaticKeyword, new List<TypeParameter>(), new List<Formal>(), new List<Formal>(), new List<AttributedExpression>(),
+      reveal = new TwoStateLemma(m.RangeToken, m.NameNode.Prepend("reveal_"), true, new List<TypeParameter>(), new List<Formal>(), new List<Formal>(), new List<AttributedExpression>(),
```

Comment:
Please also write a test to cover twostate opaque predicates too.

Reply:
Done.